### ISSUE TITLE
[Feature Request] specify which note types to check for duplicates for

### ISSUE THREAD

**Issue by hutan9013** [FooSoft/yomichan#2025]
currently, yomichan can check for duplicates either within the scope of the same note type or across all note types.
it'd be nice if there was a feature where we could specify which note types yomichan would check for duplicates.

I've also noticed that words with the same reading but different kanji are also treated as a duplicate. 
For example, if I'm tyring to add a card for 登る but のぼる already exists, it'd treat 登る as a duplicate even though のぼる might be referring to 上る or 昇る. Maybe we can also have an option to only treat words as duplicates only if they use the same kanji?

**Comment by toasted-nutbread** [comment]
> I've also noticed that words with the same reading but different kanji are also treated as a duplicate.

This is a limitation of Anki, not Yomichan. Anki treats cards which have the same value for the first field as duplicates, so if the reading is the first field, terms with the same reading will be treated as duplicates.

**Comment by hutan9013** [comment]
Oh yeah, I guess I got a bit confused there.

Regarding the 'view added note' button btw, it seems to work only if the card was added with the same note type as the current yomichan profile?, i.e. I think it still detects the card as a duplicate (green/add button greyed-out) but the 'view added note' button isn't there. I think it'd be useful to have it even for cards created with a different note type. 

Sometimes I just want to check what card I created for a specific vocab and maybe decide there if I could replace the card with a better one.
